### PR TITLE
Bugfix/admin action reset applimits

### PR DESCRIPTION
**Pull request by hamzaimran08**
Added an admin action to be able to reset app limits to default values in projects. This needs to be run from within the admin panel. Fixed small issue with paths (trailing slash needed to be removed) and configured copy commands in custom apps chart to handle empty directory. Also fixed issue [SS-912](https://scilifelab.atlassian.net/browse/SS-912).

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [ ] I have updated the release notes (releasenotes.md)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

File: charts/apps/custom-app/chart/templates/deployment.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ spec:
         command:
           - /bin/sh
           - -c
-          - cp -r -n {{ $.Values.appconfig.path }}/* /tmp
+          - if [ -n "$(ls -A {{ $.Values.appconfig.path }})" ]; then echo "Copying data..." && cp -r -n {{ $.Values.appconfig.path }}/* /tmp && echo "Done Copying"; elif [ -d "{{ $.Values.appconfig.path }}" ]; then echo "Empty directory. Nothing to copy."; else echo "Directory does not exist" && cp -r -n {{ $.Values.appconfig.path }}/* /tmp;  fi
```
Comment (sandstromviktor) [inline charts/apps/custom-app/chart/templates/deployment.yaml:51]: Lets rewrite this using the `|` operator. This is hard to read. 

```
  - |
    if [ -n "$(ls -A {{ $.Values.appconfig.path }})" ]; then
      echo "Copying data..." && cp -r -n {{ $.Values.appconfig.path }}/* /tmp && echo "Done Copying";
    elif [ -d "{{ $.Values.appconfig.path }}" ]; then
      echo "Empty directory. Nothing to copy.";
    else
      echo "Directory does not exist" && cp -r -n {{ $.Values.appconfig.path }}/* /tmp;
    fi
```

File: projects/admin.py [hunk 2]
```diff
@@ -27,6 +28,11 @@
 class ProjectAdmin(admin.ModelAdmin):
     list_display = ("name", "owner", "status", "updated_at")
     list_filter = ["owner", "status"]
+    actions = ["update_app_limits"]
+
+    @admin.action(description="Reset app limits")
+    def update_app_limits(self, request, queryset):
+        queryset.update(apps_per_project=settings.APPS_PER_PROJECT_LIMIT)
```
Comment (sandstromviktor) [inline projects/admin.py:35]: NICE!

**Review by sandstromviktor — APPROVED** [review]
Looks good! I would like to fix the 5 level deep if-nest-of-doom but we can do that later. 
I proposed a change to the custom app chart, but that is for tidyness only. We can do that later aswell.

File: charts/apps/custom-app/chart/templates/deployment.yaml (hunk 1 above)
Comment (hamzaimran08) [reply charts/apps/custom-app/chart/templates/deployment.yaml]: Done!

**Pull request merged**